app-frontend/src/app/components/common/navBar/navBar.controller.js

```diff
@@ -1,27 +1,122 @@
 
+const Plyr = require('plyr/dist/plyr.js');
+
 export default class NavBarController {
     constructor( // eslint-disable-line max-params
-        $state, APP_CONFIG, authService
+        $rootScope, $scope, $log, $state, $document, $sce, $timeout, APP_CONFIG, authService
     ) {
         'ngInject';
         if (APP_CONFIG.error) {
             this.loadError = true;
         }
-
+        this.$rootScope = $rootScope;
+        this.$scope = $scope;
+        this.$log = $log;
         this.$state = $state;
+        this.$document = $document;
+        this.$sce = $sce;
+        this.$timeout = $timeout;
         this.authService = authService;
     }
 
     $onInit() {
         this.optionsOpen = false;
         this.assetLogo = assetLogo;
+
+        this.HELPCONFIG = HELPCONFIG;
+        this.helpDocs = this.HELPCONFIG ? this.HELPCONFIG.HELP_DOCS : {};
+        if (!_.isEmpty(this.helpDocs)) {
+            this.showHelpCenter = true;
+            this.setRootDocs();
+            this.setPageDocs(this.$state);
+            this.$rootScope.$on('$stateChangeSuccess', () => {
+                this.setPageDocs(this.$state);
+            });
+        }
+    }
+
+    onHelpVideoClicked(isRootVid, doc) {
+        this.onVideoClose();
+        this.$timeout(() => {
+            this.isRootVid = isRootVid;
+            this.videoUrl = this.$sce.trustAsResourceUrl(doc.link + '?rel=0&controls=0&showinfo=0');
+            this.initPlyr();
+        }, 100);
+    }
+
+    initPlyr() {
+        this.$timeout(() => {
+            this.plyr = new Plyr('#player');
+            this.plyr.on('enterfullscreen', this.onEnterFullscreen.bind(this));
+            this.plyr.on('exitfullscreen', this.onExitFullscreen.bind(this));
+            this.showVideo = true;
+        }, 100);
+    }
+
+    onEnterFullscreen() {
+        const plyrContainer = angular.element(this.$document[0].querySelector('#player'));
+        plyrContainer.toggleClass('full-screen', true);
+        this.isFullScreenMode = true;
+        this.$scope.$evalAsync();
+    }
+
+    onExitFullscreen() {
+        const plyrContainer = angular.element(this.$document[0].querySelector('#player'));
+        plyrContainer.toggleClass('full-screen', false);
+        this.isFullScreenMode = false;
+        this.$scope.$evalAsync();
+    }
+
+    onVideoClose(e = {}) {
+        if (!_.isEmpty(e)) {
+            e.stopPropagation();
+        }
+        delete this.videoUrl;
+        this.showVideo = false;
+        this.showMini = false;
+    }
+
+    onVideoCollapse() {
+        this.showVideo = false;
+        this.$timeout(() => {
+            this.showMini = true;
+            this.isMiniVideoHover = false;
+        }, 300);
+    }
+
+    onMiniVideoHover(status) {
+        this.isMiniVideoHover = status;
+    }
+
+    onMiniVideoClick() {
+        if (this.videoUrl) {
+            this.showVideo = true;
+            this.showMini = false;
+        }
+    }
+
+    setRootDocs() {
```

Comment:
this is only used once and it's a single line, might as well inline it in the init method